wandb/sdk/launch/builder/build.py

```diff
@@ -188,8 +188,7 @@ def get_base_setup(
     CPU version is built on python, GPU version is built on nvidia:cuda.
     """
     python_base_image = f"python:{py_version}-buster"
-    if launch_project.cuda:
-        cuda_version = launch_project.cuda_version or DEFAULT_CUDA_VERSION
+    if launch_project.cuda_base_image:
```

Comment:
Can we do like at least an internal debug logging message when know we are going to try to use Cuda? maybe even a term log depending on how intrusive you think it is.

Reply:
Good call